phylogenetic/rules/prepare_sequences.smk

```diff
@@ -74,16 +79,45 @@ rule filter:
             --output-sequences {output.sequences} \
             --output-metadata {output.metadata} \
             --exclude {input.exclude} \
+            {params.exclude_where} \
             --min-date {params.min_date} \
             --min-length {params.min_length} \
             --query "(QC_rare_mutations == 'good' | QC_rare_mutations == 'mediocre')" \
             --output-log {output.log}
         """
 
 
-rule subsample:
+# At this point we merge in private data (iff requested)
+rule add_private_metadata:
+    """
+    This rule presupposes the presence of private data in the correct format
+    """
     input:
+        sequences=build_dir + "/{build_name}/good_sequences.fasta",
         metadata=build_dir + "/{build_name}/good_metadata.tsv",
+    params:
+        private_sequences="data/sequences-private.fasta",
+        private_metadata="data/metadata-private.tsv",
```

Comment:
I would make these inputs instead of params, so Snakemake can alert users immediately when they run the workflow and those files are missing.

Reply:
Of course! Switched to this usage. Very nice